pom.xml

```diff
@@ -215,14 +215,21 @@ THE SOFTWARE.
         <version>${slf4jVersion}</version>
       </dependency>
       <dependency>
+        <groupId>commons-logging</groupId>
+        <artifactId>commons-logging</artifactId>
+        <version>1.1.3</version>
+        <scope>provided</scope><!-- by jcl-over-slf4j -->
```

Comment:
@jtnord, we have used this technique successfully for years in our department pom. By specifying commons-logging as provided, you do not have to exclude it in every other current dependency *and* inhibit that it creeps in by new versions/dependencies which might introduce commons-logging as transitive dependency.  One could argue about the location, though: it would even be better to include this is in `org.jenkins-ci:jenkins`.